charts/cargohold/templates/tests/cargohold-integration.yaml

```diff
@@ -17,4 +17,12 @@ spec:
     volumeMounts:
     - name: "cargohold-integration"
       mountPath: "/etc/wire/integration"
+    env:
+    # these dummy values are necessary for Amazonka's "Discover"
+    - name: AWS_ACCESS_KEY_ID
+      value: "dummy"
+    - name: AWS_SECRET_ACCESS_KEY
+      value: "dummy"
+    - name: AWS_REGION
+      value: "eu-west-1"
```

Comment:
This is what we also do for the other services, but unless I'm missing something it is completely unnecessary?
The integration test binaries don't talk to AWS directly, so they should not need these variables.

We should probably remove them (from all integration pods, in a separate PR).

Reply:
I cannot remember right now but... amazonka might complain/throw errors if they are not defined?

Reply:
But this pod only runs the `amazonka-integration` binary, right? And that doesn't depend on amazonka.

Reply:
I also verified this locally for gundeck (from where I copied the config). I can run the binary without providing the environment variables.

Well, for now I'll merge this and then I can make a separate PR removing the config from all `-integration` pods.

Reply:
`cargohold-integration` depends on `cargohold` lib which does have a dependency on `amazonka`. I can't tell for sure if it's a problem or not; it may be different for other services because `brig-integration` and `galley-integration` do talk directly to AWS (they check the queues)